### ISSUE TITLE
[Warrior, Spell] Sweeping Strikes works good in one way.

### ISSUE THREAD

**Issue by BialyADI** [SunwellTracker/issues#2580]
Decription:
Sweeping Strikes doing less Dmg if Main Target have more Armor/Resilience than Secondary Target.
   
How it works: 
Sweeping Strikes doing less Dmg if Main Target have more Armor/Resilience than Secondary Target.

How it should work: 
Sweeping Strikes is supposed to literally copy the Damage done on the main Target to the secondary Target, provided Damage mitigation sources (or lack thereof) are the exact same -- ergo, Armor has to be the same, if there's any Resilience, it also has to be the same. Buffs/Debuffs also count.
The Damage difference is supposed to be corresponding to the difference in Damage mitigation compared to the main Target to the secondary Target.
Ex. 1:

Main Target (MT) has 40% Armor, while Secondary Target (ST) has 30% Armor;
You hit MT for 2k;
Sweeping Strikes copies that and does 2k + 10% to ST, because ST has 10% less Armor, ergo 10% less Physical Damage mitigation (AKA more Physical Damage taken), which = 2.2k Sweeping Strikes DMG on ST -- 200 more than on MT.
Ex. 2:

MT and ST both have 40% Armor;
You hit MT for 2k;
SS copies that and does 2k to ST as well, as both MT and ST have the same exact Armor %.
Ex. 3:

MT and ST both have 40% Armor, MT has 10% less Damage taken from Resilience while ST has 0%;
You hit MT for 2k;
SS copies to ST for 2k + 10% (as ST has 0% less Damage taken from Resilience while MT has 10%, ergo 10% more Damage taken compared to MT) = 2.2k -- 200 more than MT.

This is from https://github.com/SunwellTracker/issues/issues/2407.
Basically SS works good, but when Maint Target have more Arrmor/Resilience then ST, ST should take more Dmg then Maint Target but they took the same Dmg.

![WoWScrnShot_112819_065231](https://user-images.githubusercontent.com/5780624/69780488-b8755b00-11ab-11ea-9289-6d1d6140dbe4.jpg)
I hit Normal Dummy for 1873 and Sweeping Strike Heroic Dummy for 1777, because Heroic has more Armor.

![WoWScrnShot_112819_065507](https://user-images.githubusercontent.com/5780624/69780615-0db16c80-11ac-11ea-8780-68c1d5f84432.jpg)
But when I Hit Heroic Dummy first and Sweeping Strike Normal Dummy Dmg is the same 937.
Normal Dummy should take more Dmg because of Armor difference, the same is in PvP on Resilience.

**Comment by Aridam** [comment]
I just went to test immediately upon seeing this.

Testing on the Dummies, I could reproduce what you said -- Heroic Dummy as MT > SS procs on any other lesser Dummy as ST. Damage was the same as done on the MT -- it wasn't scaled accordingly to the weaker ST. Did it reverse and I got intended results.

I wanted to be sure the first test wasn't just some issue with Dummies per se (even though it wouldn't make sense, but still), so I went out in the world to attack different Mobs, and also the same Mobs, but of different Levels. This is what seems to be the case:

If you attack the exact same Mobs (Armor and other Stats, Buffs/Debuffs being the same), Damage is the same -- intended.
Fighting two Level 78 Sinewy Wolves.
> Shroudsoul's melee swing hits Sinewy Wolf for 766 Physical. 
Shroudsoul's [Sweeping Strikes] hits Sinewy Wolf for 766 Physical. 

If you attack a weaker Mob and SS damages a stronger Mob, Damage is accordingly less -- intended.
Fighting a Level 77 Grove Walker and a Level 78 Grove Walker -- **L77 is Main Target.**
> Shroudsoul's melee swing hits Grove Walker for 804 Physical. 
Shroudsoul's [Sweeping Strikes] hits Grove Walker for 790 Physical. 

If you attack a stronger Mob and SS damages a weaker Mob, Damage is the same -- **not intended -- it seems the Damage "caps" and doesn't go above, preventing proper scaling of Damage.**
Fighting the same L77 and L78 Grove Walkers -- **L78 is Main Target this time.**
> Shroudsoul's [Heroic Throw] hits Grove Walker for 802 Physical. 
Shroudsoul's gains 1 extra attacks through [Sword Specialization]. 
Shroudsoul's melee swing hits Grove Walker for 1358 Physical.(Critical) 
Shroudsoul's [Sweeping Strikes] hits Grove Walker for 1358 Physical. 
Shroudsoul's [Sweeping Strikes] hits Grove Walker for 802 Physical. 

As you can see, the Damage is "capped" on the weaker ST if hitting a stronger MT -- it won't go above the original, but it should. Refer to my original report on it to see.

Confirmed.

Blegh, what's with Warriors right now. Even past fixes are broken in one way or another at the moment.

**Comment by BialyADI** [comment]
I've been playing here for 10 years with breaks.
Always same story. Repair one, destroy another and naw we will wait months for fix.
They do nice work, frequent updates, but plz focus on repair classes. Every Skill/Talent should work perfectly on server like this.

**Comment by Riztazz** [comment]
That's how programming work when you actually work on the server, there is regress - yes but it will get fixed as well.
We prioritize classes but we've got way bigger issues on our hands right now but i can say we're almost in the clear with those and are getting back to classes.
This project (run by us) isn't even 5 years old..

**Comment by BialyADI** [comment]
If you can look on this thread https://sunwell-community.com/index.php?/topic/13771-warrior-bugs/

I know you are doing a good job, that's why I people play here for so long.
This project isn't even 5 years, but before this project Pyssy and Xinef doing well too.

**Comment by BialyADI** [comment]
Nice, https://github.com/SunwellTracker/issues/issues/2507 last thing about Sweeping Strikes.

**Comment by Riztazz** [comment]
Awesome, closing this one and will take a look at the other issue later
